package/MDAnalysis/analysis/pca.py

```diff
@@ -46,7 +46,8 @@
 explains. Stored in :attr:`PCA.cumulated_variance`, a ratio for each number of
```

Comment:
```suggestion
explains. Stored in :attr:`PCA.results.cumulated_variance`, a ratio for each number of
```
? (it'll need fixes for the number of words since I think this suggestion is > 79 chars

Reply:
You're right, well spotted. I'll fix it without using the suggestion.